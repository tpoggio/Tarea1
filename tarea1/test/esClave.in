esClave 0
esClave 15
Fin