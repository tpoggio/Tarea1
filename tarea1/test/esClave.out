1>0 no es clave. 
2>15 no es clave. 
3>Fin.
